src/app/access-requests/features/access-request-duration-edit/access-request-duration-edit.component.html

```diff
@@ -1,105 +1,82 @@
 @if (!locked()) {
-  <tr>
-    <th
-      class="w-36 min-w-36 pe-1 text-start"
-      [class.align-top]="!isOpen()"
-      [class.align-center]="isOpen()"
-    >
-      Access Starts:
-    </th>
-    <td
-      class="flex min-w-60"
-      [class.items-center]="isOpen()"
-      [class.items-start]="!isOpen()"
-    >
-      @if (isOpen()) {
-        <mat-form-field class="grow">
-          <mat-datepicker-toggle
-            matIconSuffix
-            [for]="datepickerFrom"
-          ></mat-datepicker-toggle>
-          <input
-            matInput
-            [formControl]="fromFormControl"
-            [matDatepicker]="datepickerFrom"
-            [(ngModel)]="fromField"
-            [min]="minFromDate"
-            [max]="maxFromDate"
-            (dateChange)="onDateSelected($event, true)"
-            class="pr-8"
-          />
-          @if (fromFieldErrorMessage()) {
-            <mat-error>
-              {{ fromFieldErrorMessage() }}
-            </mat-error>
-          }
-          <mat-datepicker #datepickerFrom></mat-datepicker>
-        </mat-form-field>
-        <mat-chip
-          class="smaller-chip interactive-chip cancel-edit-button mx-2 mb-5"
-          (click)="cancel()"
-          ><mat-icon>close</mat-icon></mat-chip
-        >
-        <mat-chip
-          class="smaller-chip interactive-chip save-edit-button mb-5"
-          (click)="save()"
-          [disabled]="saveDisabled()"
-          ><mat-icon>check</mat-icon></mat-chip
-        >
-      } @else {
+  @if (isOpen()) {
+    <tr>
+      <th class="w-36 min-w-36 pb-5 text-start">Access duration:</th>
+      <td class="min-w-60">
```

Comment:
```suggestion
      <td class="min-w-60 pb-1">
```

otherwise the error message is too close to the next row